content/tutorials/tutorials.json

```diff
@@ -0,0 +1,70 @@
+{
+
+  "favorites": {
+    "title": "No results for the filter",
+    "desc": "However we do have these exceptional examples of tutorials that might help you.",
+    "filenames": [
+        "publish-subscribe.textile",
+        "token-authentication.textile",
+        "_multiplayer-vr-web.textile",
+        "mqtt-snake.textile",
+        "_live-flight-tracking.textile",
+        "_multi-lingual-chat.textile"
+    ]
+  },
+
+  "categories": [
+    {
+      "id": "channels",
+      "icon": "arrows-horizontal",
+      "name": "Channels",
+      "desc": "A selection of tutorials demonstrating the use of Realtime and/or REST libraries within devices such as browsers, desktops, mobiles or servers."
+    },
+
+  "categories": [
```

Comment:
Looks like this bit is repeated (which might be causing the PR deployment to fail)

Reply:
done